common/achievement/index.ts

```diff
@@ -168,6 +169,8 @@ async function rewardUser<ID extends ActionID>(evaluationResult: number | null,
 
     const { type, group, groupOrder } = updatedAchievement.template;
 
+    // We want to delay the notification by one minute
+    const triggerNotificationAt = moment().add(1, 'minute').toDate();
```

Comment:
@Jonasdoubleyou Does this work with user-app notifications?

Reply:
It seems like it will be delivered during the next `checkReminders` schedule, which is every ten minutes...

Reply:
exactly